cylc/flow/task_pool.py

```diff
@@ -1818,7 +1820,7 @@ def spawn_task(
                 # revive as incomplete.
                 msg = "incomplete"
 
-            if cylc.flow.flags.verbosity >= 1:
+            if LOG.level == logging.DEBUG:
```

Comment:
Log level does not currently affect verbosity in tests. (This change is correct anyway, because the code block applies specifically to debug log level).

Reply:
The log level is an integer, `logging.DEBUG = 10`, `logging.INFO = 20`, etc.

The handler level can be set to any arbitrary integer:

```suggestion
            if LOG.level <= logging.DEBUG:
```